app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] @BB-Simon and @MNisarAli Great work setting up the devisee for authentication, however, you forgot to add `:confirmable` feature that allows  application to asks for confirmation of email after they register.

Reply:
Hi @danmainah 
Thanks for your review, I have done the required.